rgw/v2/tests/s3_swift/reusables/bucket_notification.py

```diff
@@ -149,23 +150,45 @@ def get_topic(client, topic_arn, ceph_version):
             log.info(f"get topic attributes: {get_topic_info_json}")
 
 
-def rgw_admin_topic_notif_ops(op, args, sub_command="topic"):
+def rgw_admin_topic_notif_ops(config, op, args, sub_command="topic"):
```

Comment:
@viduship 
I think below call to this method also needs to be changed
https://github.com/red-hat-storage/ceph-qe-scripts/blob/master/rgw/v2/tests/s3_swift/test_bucket_policy_with_tenant_user.py#L229

please change it there also..